src/Microsoft.ML.Data/Evaluators/Metrics/MultiClassClassifierMetrics.cs

```diff
@@ -17,7 +17,7 @@ public sealed class MultiClassClassifierMetrics
         /// The micro-average metric weighs each class according to the number of instances that belong
         /// to it in the dataset.
```

Comment:
The explanation in this summary is written differently than the one for MacroAccuracy. Can you please rewrite them to be similar?

Reply:
Can you please dictate?

---
In reply to: [259550663](https://github.com/dotnet/machinelearning/pull/2700#discussion_r259550663) [](ancestors = 259550663)